<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogScheduleMovie</class>
 <widget class="QDialog" name="DialogScheduleMovie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seleccionar Pelicula</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="dias">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(187, 218, 255);</string>
     </property>
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="Lunes">
      <attribute name="title">
       <string>Lunes</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="ListaLunes">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>615</width>
            <height>368</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QListWidget" name="listLunes"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Martes">
      <attribute name="title">
       <string>Martes</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>615</width>
            <height>368</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QListWidget" name="listMartes"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Miercoles">
      <attribute name="title">
       <string>Miercoles</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>615</width>
            <height>368</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QListWidget" name="listMiercoles"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Jueves">
      <attribute name="title">
       <string>Jueves</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea_4">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>615</width>
            <height>368</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QListWidget" name="listJueves"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Viernes">
      <attribute name="title">
       <string>Viernes</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QScrollArea" name="scrollArea_5">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>615</width>
            <height>368</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QListWidget" name="listViernes"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Sabado">
      <attribute name="title">
       <string>Sabado</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QScrollArea" name="scrollArea_6">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>615</width>
            <height>368</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QListWidget" name="listSabado"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Domingo">
      <attribute name="title">
       <string>Domingo</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QScrollArea" name="scrollArea_7">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>615</width>
            <height>368</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QListWidget" name="listDomingo"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Doble click para seleccionar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
